inst/tinytest/test_loneliness_scotland_iz22.R

```diff
@@ -0,0 +1,107 @@
+library(geographr)
+library(dplyr)
+
+# pkgload::load_all(".")
+
+# Load dataframe
+# Use dummy dataset for now; to be changed when actual data is pushed
+data(dummy_loneliness_scotland_iz22)
+
+# ---- Tests: Overall dataframe ----
+# Test its class as a tibble data frame
+expect_equal(
+  class(dummy_loneliness_scotland_iz22)[1],
+  "tbl_df"
+)
+
+# Test its dimensions: 1279 interzones, and 4 columns
+expect_equal(
+  dim(dummy_loneliness_scotland_iz22),
+  c(1279, 4)
+)
+
+# ---- Tests: Interzone ----
+# Test class
+expect_equal(
+  class(dummy_loneliness_scotland_iz22$iz_code11),
+  "character"
+)
+
+# Test all Interzone codes are there
+scotland_iz_codes <-
+  lookup_dz11_iz11_ltla20 |>
```

Comment:
This line appears to be the only time the `geographr` is called. Considering the package isn't yet released to CRAN, isn't stable, and is large, it doesn't make sense to add a dependency to the package, if avoidable.

Instead, can we store a list of IZ codes somewhere, and read those in, instead? We could place these in `inst/extdata/` for example.